bot/plugins/instance.py

```diff
@@ -395,41 +469,69 @@ async def execute(self) -> None:
             await plugin.app.rest.trigger_typing(self.channel)
 
         # try to execute code
-        att: hikari.Bytes | None = None
-        out: str | None
+        code_attr: hikari.Bytes | None = None
+        stdin_attr: hikari.Bytes | None = None
```

Comment:
This was probably done out of habit but these should be `att` or `attachment`, rather than `attr` (which stands for "attribute").

A better name would probably be `code_file` and `stdin_file` though.

Reply:
changed